library/src/main/kotlin/com/thefuntasty/donut/DonutProgressView.kt

```diff
@@ -164,44 +176,79 @@ class DonutProgressView @JvmOverloads constructor(
     }
 
     /**
-     * Submits new data set to the view.
-     *
-     * New progress line will be created for each non-existent entry category and view
-     * will be animated to new state.
+     * Returns current data.
+     */
+    fun getData() = data.toList()
+
+    /**
+     * Sets color of lines belonging to specified [category] to provided [color].
+     */
+    fun setColor(category: String, @ColorInt color: Int) {
```

Comment:
What about to create `dataset` class with definition of color and entries? 🤔

Reply:
Yeah, that's on option, too. I will look into that.